README.md

```diff
@@ -13,6 +13,10 @@ Tasks
 * `conjure`: runs Conjure generation. Runs for all of the entries specified in the configuration in order. The working
   directory is set to be the project directory.
 * `conjure-publish`: publishes IR to a specified destination.
+* `conjure-backcompat`: checks backward compatibility of Conjure APIs by comparing the current IR against the previously
+  published version. Runs automatically as part of `./godelw verify`. Projects without a `group-id` or with `publish: false`
+  are skipped.
+* `conjure-accept-backcompat-breaks`: accepts backward compatibility breaks by writing acknowledgment entries to lockfiles.
```

Comment:
```suggestion
* `conjure-accept-backcompat-breaks`: accepts any compatibility breaks identified by `conjure-backcompat` (if supported by the asset).
```

Similarly to the above, the generic contract at the plugin level is "accepts any compatibility breaks identified by `conjure-backcompat` (if supported by the asset)."

Probably fine to include this, although now that this is written out another possibility would be for an asset to specify whether or not it supports the concept of "accepting" backcompat breaks.